pkg/controller/config.go

```diff
@@ -134,7 +134,7 @@ type ControllerConfig struct {
 	PodIpPool []ipam.IpRange `json:"pod-ip-pool,omitempty"`
 
 	// The number of IP addresses to allocate when a pod starts to run low
-	PodIpPoolChunkSize int `json:"pod-ip-pool-chunk-size,omitempty"`
+	PodIpPoolChunkSize int `json:"pod-subnet-chunk-size,omitempty"`
```

Comment:
Also, what happens if this configuration is not present. I know you have instrumented acc-provision to always populate this but what if the config files gets updated later. There should be a default here as well (if not set).

Reply:
Included in latest commit